### ISSUE TITLE
put /player/queue/order (moveEntry) doesn't seem to work

### ISSUE THREAD

**Issue by IIIuminator** [BjoernPetersen/MusicBot-desktop#46]
Queries return the queue as expected but the queue isn't changed server-side. Server logs show no errors.

#### Example body:
```json
{
   "song":{
      "id":"Tt7zgalyqn4wqxsj234bnxdwcwi",
      "title":"Livin' On A Prayer",
      "description":"Bon Jovi",
      "albumArtUrl":"http://lh3.googleusercontent.com/PZMktr0qSdezNLkmre2ayylJlPxyHbwaqF4evXcJQxQ2IC48W0ycyI3Rv6WVD9ckdqv4rn6J6gA",
      "duration":250,
      "provider":{
         "id":"com.github.felixgail.musicbot.gplaymusic.GPlayMusicProviderBase",
         "name":"GPlayMusic"
      }
   },
   "userName":"ivo"
}
```

#### Corresponding Query URL:
`PUT /player/queue/order?index=6`

**Comment by BjoernPetersen** [comment]
Fixed by 775d7a29c72dcd06c7db2d0addc376d10ff6ded9 .

Note that a future version of the API probably won't expect a full `QueueEntry` object here anymore to prevent such issues (BjoernPetersen/MusicBot-API#9).

**Comment by BjoernPetersen** [comment]
This seems to still be an issue with long queues

**Comment by BjoernPetersen** [comment]
Fixed by a975968d5d0ac84f9a2ee4c6a99f2b4fd548c06b
